packages/express-did-auth/src/types.ts

```diff
@@ -0,0 +1,102 @@
+import { JWTPayload, Signer } from 'did-jwt'
```

Comment:
Is this a good approach? I mean, the file with all the types. I really don't know, but, I would have put the types on the same ts file it is being implemented.

Reply:
Already extracted the types that are strongly related to one implementation, I left there the one that are shared within the entire package